index.html

```diff
@@ -8,11 +8,144 @@
 
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
         integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    
+    <link rel="stylesheet" href="./styles/style.css">
 </head>
 <body>
+    <header>
+        <div class="top-header">
+            <img src="./images/icons/facebook.png" alt="facebook icon">
+            <img src="./images/icons/twitter.png" alt="">
```

Comment:
- [ ] Kindly fix the image path because the image directory is `Images` not images or change the name of directory to affect in all website instead change of each image path